Resources/Private/Partials/Columns/Group.html

```
@@ -0,0 +1,38 @@
+                        <a
+                            href="#"
+                            title="{f:translate(key:'LLL:EXT:xima_typo3_recordlist/Resources/Private/Language/locallang.xlf:table.button.edit')}"
+                            data-table="{tableName}"
+                            data-uid="{fieldValue.value}"
+                            class="badge badge-pill group-badge {f:if(condition: '{colConfig.filter.value} == {combinedValue}', then: ' badge-primary')}"
+                            data-workspaceId="{workspaceId}"
+                            data-field-name="{colConfig.columnName}">
+                            <f:if condition="{colConfig.filter.tables.{tableName}.iconIdentifier}">
+                                <core:icon identifier="{colConfig.filter.tables.{tableName}.iconIdentifier}" size="small" />
+                            </f:if>
+                            {fieldValue.label->f:format.crop(maxCharacters: 20)}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)